src/main/java/com/liferay/faces/test/selenium/Browser.java

```diff
@@ -133,6 +143,103 @@ public static Browser getInstance() {
 		return instance;
 	}
 
+	/**
+	 * Calls {@link #captureCurrentPageState(java.lang.String, java.lang.String)}. The default output directory is
+	 * <code> {@link TestUtil#JAVA_IO_TMPDIR} + "selenium"</code>. This can be changed with the <code>
+	 * "integration.selenium.output.directory"</code> system property. No file name prefix is used.
+	 *
+	 * @see  {@link #captureCurrentPageState(java.lang.String, java.lang.String)}
+	 */
+	public void captureCurrentPageState() {
+
+		String outputDirectoryPath = TestUtil.getSystemPropertyOrDefault("integration.selenium.output.directory",
+				TestUtil.JAVA_IO_TMPDIR + "selenium");
+		captureCurrentPageState(outputDirectoryPath, null);
+	}
+
+	/**
+	 * Captures the current page markup (using {@link #getCurrentPageMarkup()}) to a file, and if the browser supports
+	 * it, captures a screenshot to a file as well. This method logs the file locations for the benefit of the tester.
```

Comment:
@pcwhite,
cc @ngriffin7a, @vsingleton,
Please read the above JavaDoc for both `captureCurrentPageState()` methods. I added these methods as a convenience to capture screenshots (if possible) and page markup. It will likely be very helpful when debugging headless browser testing, so please familiarize yourself with it. Feel free to suggest any improvements.